framework/audio/internal/platform/lin/linuxaudiodriver.cpp

```diff
@@ -51,6 +51,7 @@ static void* alsaThread(void* aParam)
         }
     }
 
+    LOGI() << "exit";
```

Comment:
Do you still need it? Maybe LOGD

Reply:
I think LOGI is better, this is important information, it will appear once when close the application.